### ISSUE TITLE
"ignore-from-file" not taking into account gitignore exceptions

### ISSUE THREAD

**Issue by drAlberT** [adrienverge/yamllint#585]
Using `ignore-from-file .gitignore` seems to wrongly handle the case like the following:

```
.*
!.checkmake
!.editorconfig
!.gitignore
!.gitlab/
!.gitlab-ci.yml
!.gitleaks.yml
```

With the above config no error is reported in the `.gitlab-ci.yml`file, while it works perfectly removing `ignore-from-file .gitignore` directive in .yamllint.yml

Could be the problem here? Isn't expected to have lines instead of the entire file?
https://github.com/adrienverge/yamllint/blob/a68c3aa69e298ea20bdfb51ac388d5bfc4c347aa/yamllint/config.py#L113C78-L113C78

**Comment by drAlberT** [comment]
Last version seems to work properly. Closing
